apps/i18n/common/en_us.json

```diff
@@ -645,6 +645,10 @@
   "dropletBlock_mathRound_param0": "x",
   "dropletBlock_mathRound_param0_description": "An arbitrary number.",
   "dropletBlock_mathRound_signatureOverride": "Math.round(x)",
+  "dropletBlock_mathSqrt_description": "Returns the square root of x",
+  "dropletBlock_mathSqrt_param0": "x",
+  "dropletBlock_mathSqrt_param0_description": "An arbitrary number.",
```

Comment:
I added this string to follow the convention of the rest of the blocks, but I don't actually see anywhere that it's displayed/used. If anyone knows, let me know!

Reply:
I _think_ these get referenced programmatically by droplet internals; it does something like `dropletBlock_${block.docFunc}_description`, so this looks correct to me!